magnum_capi_helm/driver.py

```diff
@@ -685,6 +685,26 @@ def _update_helm_release(self, context, cluster, nodegroups=None):
             }
             values = helm.mergeconcat(values, network_details)
 
+        # CatalystCloud: K8s keystone auth webhook
+        if CONF.capi_helm.k8s_keystone_auth_enabled:
```

Comment:
I think it would be more consistent to use a template variable for this? But I am guessing this doesn't work for you use case? Would making it on by default help, because I see a strong case for that.

Reply:
Indeed I hadn't considered using a template variable because, as you mention, it doesn't fit our use case. However I am happy to change this to a template variable.

Reply:
I've changed it to a template variable and adjusted the tests and minimum helm chart version accordingly.